### ISSUE TITLE
Autocomplete items are not highlighted on mouse over or when keyboard navigation is used in Material Black theme.

### ISSUE THREAD

**Issue by nemalp** [telerik/kendo-ui-core#972]
Can be reproduced here: http://demos.telerik.com/kendo-ui/autocomplete/keyboard-navigation

Reported in support thread: 953744

**Comment by inikolova** [comment]
Not reproducible any more.
